devops/.env

```diff
@@ -0,0 +1,5 @@
+# Environment config for docker compose
+
+COMPOSE_PROJECT_NAME=upenn_contrast
+GIRDER_ADMIN_USER=admin
+GIRDER_ADMIN_PASS=letmein
```

Comment:
I'd recommend setting the password to the same as the fallback password (elsewhere it is upenn).  It doesn't matter which is the default.

Reply:
done